SD-K9/src/environments/environment.prod.ts

```diff
@@ -1,3 +1,13 @@
 export const environment = {
-  production: true
+  production: true,
+  firebase: {
+    apiKey: "AIzaSyC1bI-ntc3Yy14rhdIEhuxpnmsCvqZOL90",
```

Comment:
Is it better to keep the apiKey private?

Reply:
For firebase configuration it is safe. [_One of the many sources_](https://github.com/zooniverse/vox/issues/32), [_Another source_](https://stackoverflow.com/questions/40985134/is-it-safe-to-upload-a-project-with-firebase-database-on-github)

From what I understood, the configuration generated by firebase lets you identify which project in firebase you want to use. Security mesures and constraints are specified in [Firebase Console](https://console.firebase.google.com/). You need to use the email specified in our Slack messages to modify these constraints.